neptune/internal/api_clients/hosted_api_clients/hosted_alpha_leaderboard_api_client.py

```diff
@@ -450,9 +450,7 @@ def log_artifact(self, experiment, artifact, destination=None):
     @staticmethod
     def _get_dest_and_ext(target_name):
         qualified_target_name = f'{alpha_consts.ARTIFACT_ATTRIBUTE_SPACE}{target_name}'
-        dest_path = alpha_path_utils.parse_path(normalize_file_name(qualified_target_name))
-        ext = os.path.splitext(target_name)[1]
-        return dest_path, ext
+        return alpha_path_utils.parse_path(normalize_file_name(qualified_target_name))
```

Comment:
you shoud return dest_path - it contains the proper attribute path together with file extension (the desired behaviour)